app/models/user.rb

```diff
@@ -0,0 +1,56 @@
+class User < ApplicationRecord
+  has_secure_password
+
+  enum role: ["default", "admin"]
+
+  # belongs_to :location
+
+  # validates_presence_of :auth_token_linkedin,
```

Comment:
@katiekeel @yoasyo25 I noticed we have these User attributes in the migration and schema, and I wasn't sure if there is a reason the validations should be commented out. Is it okay to merge comments into development as long as they don't get merged to master?

Reply:
@katiekeel @yoasyo25 Did you leave them out to make the initial seed data easier to handle?

Reply:
Originally, we thought we would be able to pull user data from the LinkedIn API without a user logging in to LinkedIn, so we left those attributes on the user with default nil values. The validations being present still prevented us from getting the bcrypt solution up and running, so we commented them out for the time being. We can delete those validations and validations tests for now and re-add them later if we add those attributes back onto the user. I think we'll be able to do this if we implement some kind of button on the user profile page that allows a user to import their LinkedIn data.